src/anomalib/data/base/video.py

```diff
@@ -10,6 +10,7 @@
 import torch
 from pandas import DataFrame
 from torchvision.transforms.v2 import Transform
+from torchvision.transforms.v2.functional._misc import to_dtype_video
```

Comment:
This would also work?
```suggestion
from torchvision.transforms.v2.functional import to_dtype_video
```